httptrace/server.go

```diff
@@ -100,7 +100,7 @@ func Middleware(c appdash.Collector, conf *MiddlewareConfig) func(rw http.Respon
 		if e.Route != "" {
 			rec.Name(e.Route)
 		} else {
-			rec.Name(e.Request.Host)
+			rec.Name(r.URL.Host + r.URL.Path)
```

Comment:
Is the change from `e.Request.Host` to `r.URL.Host` intended? Just asking because it looks a little suspect.